src/SimpleSAML/SessionHandlerPHP.php

```diff
@@ -341,6 +341,15 @@ public function setCookie(string $sessionName, ?string $sessionID, ?array $cooki
             session_write_close();
         }
 
+        if (array_key_exists('expire', $cookieParams)) {
+            // Simiar to the Util\HTTP::setCookie()
```

Comment:
typo

Reply:
updated. I think the comment was a last minute thing to mention where some the `isset` part came from. Since we know it is already set, since we just checked array_key_exists.